src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -894,7 +891,7 @@ void DatabaseCatalog::enqueueDroppedTableCleanup(StorageID table_id, StoragePtr
             create->setTable(table_id.table_name);
             try
             {
-                table = createTableFromAST(*create, table_id.getDatabaseName(), data_path, getContext(), false).second;
```

Comment:
why was this changed?

Reply:
It's not related to `store/` cleanup, but it's better to set `force_restore` to `true` here to avoid unnecessary exceptions, because we don't really care about data when we load it just to drop.